Automattic/woocommerce-payments
# Enable deferred intent creation UPE for SEPA

Fixes https://github.com/Automattic/woocommerce-payments/issues/7113

#### Changes proposed in this Pull Request
After SEPA has been disabled as a reusable payment method (including Subscriptions) in https://github.com/Automattic/woocommerce-payments/pull/7107, this PR removes the fallback from the deferred intent creation UPE to legacy UPE so that SEPA can be seamlessly used with the deferred intent creation UPE. 









#### Testing instructions





_Legacy UPE_
1. Navigate to WCPay DevTools and enable legacy UPE
2. Enable SEPA in `Payments -> Settings`
3. Confirm that shortcode & Blocks checkout both work as expected when paying with SEPA for a physical product (check the order status in WC Core and Stripe Dashboard)
4. Confirm that shortcode & Blocks checkout don't list SEPA as an option for subscription-based carts nor allow saving SEPA payment method for future usage (more details in: https://github.com/Automattic/woocommerce-payments/pull/7107)
5. Enable Stripe Link and perform two payments (shortcode checkout & Blocks) to confirm that Stripe Link works as expected after changes that touched this area a bit

_Deferred intent creation UPE_
1. Navigate to WCPay DevTools and enable deferred intent creation UPE
2. Enable SEPA in `Payments -> Settings`
3. Open the shortcode checkout page and confirm that there's no fallback to the legacy UPE and instead SEPA is listed as part of the deferred UPE gateways list
4. Perform steps 3-5 from the previous testing block above on the legacy UPE

-------------------


**Post merge**
